androidTest/src/main/kotlin/dev/johnoreilly/confetti/screenshot/a11y/A11ySnapshotTransformer.kt

```diff
@@ -28,9 +28,9 @@ import androidx.compose.ui.semantics.SemanticsNode
 import androidx.compose.ui.semantics.SemanticsProperties
 import androidx.compose.ui.semantics.getOrNull
 import androidx.compose.ui.test.SemanticsNodeInteraction
-import dev.johnoreilly.confetti.wear.screenshots.SnapshotTransformer
+import dev.johnoreilly.confetti.screenshot.SnapshotTransformer
 
-class A11ySnapshotTransformer : SnapshotTransformer {
+class A11ySnapshotTransformer : dev.johnoreilly.confetti.screenshot.SnapshotTransformer {
```

Comment:
nit: fix imports

Reply:
Nice catch! Fixed it in [6d96d29](https://github.com/joreilly/Confetti/pull/528/commits/6d96d29ff90170fd86d47cb4ec181995f80acc12) :) Thanks!